pkg/sources/sources.go

```diff
@@ -266,6 +266,10 @@ type GitlabConfig struct {
 	Filter *common.Filter
 	// SkipBinaries allows skipping binary files from the scan.
 	SkipBinaries bool
+	// IncludeRepos is a list of repositories to include in the scan.
```

Comment:
Is this a list of _repositories_ or _repository glob patterns_?

Reply:
List of repository glob patterns.